bors.toml

```diff
@@ -0,0 +1,10 @@
+status = [
+  "Lint",
+  "Unit Tests (1.15)",
+  "Unit Tests (1.16)",
+  "Integration Tests (1.15)",
+  "Integration Tests (1.16)",
```

Comment:
Nit: we may be able to simplify this using [SQL LIKE](https://www.w3schools.com/sql/sql_like.asp) syntax